ORK1Kit/ORK1Kit/Common/ORK1Types.h

```diff
@@ -140,6 +140,25 @@ typedef NS_ENUM(NSInteger, ORK1ChoiceAnswerStyle) {
     ORK1ChoiceAnswerStyleMultipleChoice
 } ORK1_ENUM_AVAILABLE;
 
+/**
+ An enumeration of how to display detailText/description
+ */
+typedef NS_ENUM(NSInteger, ORK1ChoiceDescriptionStyle) {
+    /**
+     No detailText/description appears
+     */
+    ORK1ChoiceDescriptionStyleNone,
+    
+    /**
+     The detailText/description always appears under the answer choice
+     */
+    ORK1ChoiceDescriptionStyleDisplayAlways,
```

Comment:
Would it make more sense for Always, instead of None, to be the default (zero) value?

Reply:
I agree - good catch